code/modules/research/techweb/all_nodes.dm

```diff
@@ -343,12 +343,21 @@
 /datum/techweb_node/cyborg_upg_combat
 	id = "cyborg_upg_combat"
 	display_name = "Cyborg Upgrades: Combat"
-	description = "Military grade upgrades for cyborgs."
+	description = "Military-grade upgrades for cyborgs."
 	prereq_ids = list("adv_robotics", "adv_engi" , "weaponry")
 	design_ids = list("borg_upgrade_disablercooler")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
 	export_price = 5000
 
+/datum/techweb_node/cyborg_upg_syndiborg
+	id = "cyborg_upg_syndiborg"
+	display_name = "Cyborg Upgrades: Syndicate"
+	description = "Upgrades for syndicate cyborgs." //yeah, this kind of breaks the pattern of the descriptions for these things, but the illegal technology node already has a "syndicate upgrade" for cyborgs in the form of the illegal equipment module
+	prereq_ids = list("syndicate_basic", "cyborg_upg_combat")
+	design_ids = list("borg_upgrade_syndicate_surgical_tools", "borg_upgrade_syndicate_medical_sabotage")
```

Comment:
shouldnt the cyborg syndicate upgrade be moved here from illegal tech?
maybe ai surveillance disk too

Reply:
Well, the idea is that node is for upgrades specifically for syndicate cyborgs, not just syndicate-themed upgrades for Nanotrasen silicons.

Reply:
there is literally no reason for this node to exist, crew doesnt have access to syndicate borgs